libs/components/core/src/lib/modules/numeric/numeric.pipe.ts

```diff
@@ -61,7 +61,14 @@ export class SkyNumericPipe implements PipeTransform, OnDestroy {
   /**
    * Formats a number based on the provided options.
    */
-  public transform(value: number, config?: SkyNumericOptions): string {
+  public transform(
+    value: number | undefined,
+    config?: SkyNumericOptions
+  ): string {
+    if (value === undefined || isNaN(value) || value === null) {
```

Comment:
If you're going to check for `null`, I would add that to the value's type as well.
```
value: number | undefined | null
```
Similar to Angular's DatePipe: https://github.com/angular/angular/blob/e1728a2a392aa7fa237a50561469f99eb201a24c/packages/common/src/pipes/date_pipe.ts#L256

Reply:
Done. I hadn't changed it for the `null` since that was exisiting - but its a good point.